src/sapB_fmt_plug.c

```diff
@@ -198,10 +198,10 @@ static void init(struct fmt_main *self)
 		fprintf(stderr, "Warning: SAP-B format should never be UTF-8.\nUse --target-encoding=iso-8859-1 or whatever is applicable.\n");
 
 #if defined (_OPENMP)
-	omp_t = omp_get_max_threads();
-	self->params.min_keys_per_crypt = (omp_t * MIN_KEYS_PER_CRYPT);
-	omp_t *= OMP_SCALE;
-	self->params.max_keys_per_crypt = (omp_t * MAX_KEYS_PER_CRYPT);
+	threads = omp_get_max_threads();
+	self->params.min_keys_per_crypt = (threads * MIN_KEYS_PER_CRYPT);
+	threads *= OMP_SCALE;
+	self->params.max_keys_per_crypt = (threads * MAX_KEYS_PER_CRYPT);
```

Comment:
This is missing `if (threads > 1)`

Reply:
Yes, I noticed this earlier this morning (was it some other file?). I am thinking of doing a new "Assorted code cleanups, part 2" PR to address this, and similar problems.

Reply:
If you do, please make separate commits at least for your "scripted" changes.
I think it is easier to review lots of similar changes, compared to mixed changes.